.concourse/tasks/upgrade.sh

```diff
@@ -1,10 +1,40 @@
 #!/bin/bash
 
+# Notes
+# - Parameters:
+#   - GKE_KEY
+#   - GKE_PROJECT
+#   - GKE_ZONE
+#
+# - Variables used by catapult:
+#   - GKE_CLUSTER_NAME
+#   - GKE_CLUSTER_ZONE
+#   - GKE_CRED_JSON
+#   - GKE_PROJECT
+#
+#   - BACKEND
+#   - CONFIG_OVERRIDE
+#   - ENABLE_EIRINI
+#   - KUBECFG
+#   - QUIET_OUTPUT
+#   - SCF_CHART
+#   - SCF_OPERATOR
+#
+# Variables used by kubectl, gcloud, ...
+#   - KUBECONFIG
+
 set -e
+set -x
 
 echo "Running the upgrade test"
 
-KUBECF_LATEST_RELEASE=$(cat kubecf-github-release/version)
+# Fail early for missing parameters
+GKE_KEY="$GKE_KEY"
+GKE_PROJECT="$GKE_PROJECT"
+GKE_CLUSTER_ZONE="$GKE_ZONE"
+export GKE_CLUSTER_ZONE
```

Comment:
```suggestion
export GKE_CLUSTER_ZONE="${GKE_ZONE:?}"
```

There isn't a process substitution here, no need to put `export` on a separate line.

Reply:
Ok. Is that confirmed by the linter ?

Reply:
Done with commit 54507d8b51b33626385ae6769c4abdc6b8edcd11